detection_rules/etc/stack-schema-map.yaml

```diff
@@ -107,4 +107,9 @@
 "8.15.0":
   beats: "8.13.4"
   ecs: "8.11.0"
+  endgame: "8.4.0"
+
+"8.16.0":
+  beats: "8.14.3"
+  ecs: "8.11.0"
```

Comment:
These look off. Shouldn't they be a later version?

Reply:
Beats is updated to latest available. I have double checked ecs is 8.11.0 only https://github.com/elastic/ecs/tags

Reply:
Yep, we only reference what is available via release tags.

Reply:
Yea I guess these shouldn't be changing that much 🤷